components/callbacks.py

```diff
@@ -198,24 +196,25 @@ def off_on_topic(update: Update, context: CallbackContext) -> None:
                 else:
                     name = reply.from_user.first_name
             else:
+                # Probably never happens anyway ...
                 name = "Somebody"
 
             replied_message_text = reply.text_html
             replied_message_id = reply.message_id
 
             text = (
-                f'{name} <a href="t.me/pythontelegrambotgroup/{replied_message_id}">wrote</a>:\n'
+                f'{name} <a href="t.me/{ONTOPIC_USERNAME}/{replied_message_id}">wrote</a>:\n'
                 f"{replied_message_text}\n\n"
                 f"⬇️ ᴘʟᴇᴀsᴇ ᴄᴏɴᴛɪɴᴜᴇ ʜᴇʀᴇ ⬇️"
             )
 
             offtopic_msg = context.bot.send_message(OFFTOPIC_CHAT_ID, text)
 
             message.reply_text(
-                moved_notification.format(
-                    "https://telegram.me/pythontelegrambottalk/" + str(offtopic_msg.message_id)
+                text=(
+                    'I moved this discussion to the <a href="https://t.me/'
+                    f'{OFFTOPIC_USERNAME}/{offtopic_msg.message_id}">off-topic group</a>.'
                 ),
-                parse_mode=ParseMode.MARKDOWN,
```

Comment:
um should be changed to Html or use reply_html?

Reply:
We use `Defaults(parse_mode=ParseMode.HTML, disable_web_page_preview=True)` :)